src/v/cluster/node/types.cc

```diff
@@ -55,6 +68,11 @@ void local_state::serde_read(iobuf_parser& in, const serde::header& h) {
             d.alert = storage_space_alert;
         }
     }
+
+    if (h._version >= 1) {
+        log_data_size = serde::read_nested<std::optional<log_data_state>>(
+          in, 0);
+    }
```

Comment:
Since we've already branched out shouldn't we bump the serde version here?

Reply:
I fixed this